### PR TITLE
build: upgrade angular 11

### LINKED ISSUE #121: @angular/core ^11.0.0 release / support
- update peer dependencies.
- implement necessary changes (if any)
- move `<11.0.0` to support branch.

### PR DESCRIPTION
**Pull request by damingerdai**
Updated Angular to version 11

close #121

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -215,6 +218,7 @@
   },
   "defaultProject": "app",
   "cli": {
+    "analytics": "2103a9ef-0556-42fe-ba20-95467c2416b0",
```
Comment (bbortt) [inline angular.json:221]: what is this for?

File: README.md [hunk 2]
```diff
@@ -5,7 +5,9 @@ This is the main branch for @angular/material 10.x
 You can find the Angular 5 version on the `1.x` branch. \
 You can find the Angular 6 version on the `2.x` branch. \
 You can find the Angular 8 version on the `3.x` branch. \
-You can find the Angular 9 version on the `4.x` branch.
+You can find the Angular 9 version on the `4.x` branch. \
+You can find the Angular 10 version on the `5.x` branch. \
+You can find the Angular 11 version on the `6.x` branch. \
```
Comment (bbortt) [inline README.md:10]: actually I wouldn't open the support branch for the current release as long as it is active ;)

File: README.md [hunk 3]
```diff
@@ -5,7 +5,9 @@ This is the main branch for @angular/material 10.x
 You can find the Angular 5 version on the `1.x` branch. \
 You can find the Angular 6 version on the `2.x` branch. \
 You can find the Angular 8 version on the `3.x` branch. \
-You can find the Angular 9 version on the `4.x` branch.
+You can find the Angular 9 version on the `4.x` branch. \
+You can find the Angular 10 version on the `5.x` branch. \
```
Comment (bbortt) [inline README.md:9]: !remindme

**Review by bbortt — CHANGES_REQUESTED** [review]
could you please align the versions to `6.0.0` as well? `package.json` in root, in projects and the `peerDependency` in `moment`. that would be great!

**Comment by damingerdai** [comment]
@bbortt please review agagin

File: src/app/app.component.spec.ts [hunk 4]
```diff
@@ -1,25 +1,25 @@
-import {async, TestBed} from "@angular/core/testing";
+import { TestBed, waitForAsync } from "@angular/core/testing";
```
Comment (bbortt) [inline src/app/app.component.spec.ts:1]: import does not exactly match format. let's overlook that..

**Review by bbortt — APPROVED** [review]
thx very much! will release it later this day.

File: src/app/app.component.spec.ts (hunk 4 above)
Comment (damingerdai) [reply src/app/app.component.spec.ts:1]: how adbout the tslint rule:
```
"ordered-imports": [
			true,
			{
				"import-sources-order": "lowercase-last",
				"named-imports-order": "lowercase-first"
			}
],
```

Comment (bbortt) [reply src/app/app.component.spec.ts:1]: I meant the bracket spacing.. but nvm it! 👍🏼

**Comment by damingerdai** [comment]
@bbortt please review. thanks

**Comment by bbortt** [comment]
introduced in #126.

**Pull request closed without merging**